pyproject.toml

```diff
@@ -1,3 +1,43 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pyjanitor"
+version = "0.26.0"
+description="Tools for cleaning pandas DataFrames"
+url="https://github.com/pyjanitor-devs/pyjanitor"
+license = {text = "MIT License"}
+requires-python = ">= 3.8"
+readme = 'README.md'
+dependencies = [
+    "pandas >= 2.0.0",
+    "pandas_flavor",
+    "multiple_dispatch",
+    "scipy"
+]
```

Comment:
Wow! Is the dependency list really this small? I suspect it might be longer, though.

Reply:
i pulled it from `.requirements/base.in', based on information in setup.py